services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationControllerITest.kt

```diff
@@ -1,12 +1,10 @@
 package fi.hel.haitaton.hanke.application
 
 import com.fasterxml.jackson.databind.node.ObjectNode
-import fi.hel.haitaton.hanke.ControllerTest
-import fi.hel.haitaton.hanke.IntegrationTestConfiguration
-import fi.hel.haitaton.hanke.OBJECT_MAPPER
-import fi.hel.haitaton.hanke.andReturnBody
+import fi.hel.haitaton.hanke.*
```

Comment:
I don't think there's any reason to use wildcard imports these days. The only benefit they give is less lines of code, but IDEs hide them by default. When not using and IDE, like here in Github, it's better to see clearly where classes are imported from. I think this is further highlighted in Kotlin, where there's no difference between importing classes and importing functions and extension functions.

Reply:
Agreed. My auto import settings for Kotlin were not configured correctly.